shared/js/sww/_register.js

```diff
@@ -0,0 +1,68 @@
+function installServiceWorker() {
```

Comment:
I hope that we don't have to worry about sw installation once bug 1156198 lands.